contracts/src/test/kotlin/com/r3/developers/csdetemplate/utxoexample/contracts/ChatContractCreateCommandTest.kt

```diff
@@ -35,6 +38,34 @@ class ChatContractCreateCommandTest : ContractTest() {
         assertVerifies(transaction)
     }
 
+    @Test
+    fun addAttachmentsNotSupported() {
+        val secureHash: SecureHash = object : SecureHash {
+            override fun getAlgorithm(): String {
+                return null.toString()
+            }
+
+            override fun toHexString(): String {
+                return null.toString();
+            }
+        }
+        val exception: Exception = Assertions.assertThrows(
+            UnsupportedOperationException::class.java
+        ) {
+            val transaction = ledgerService
+                .createTransactionBuilder()
+                .addAttachment(secureHash)
+                .addOutputState(outputChatState)
+                .addCommand(Create())
+                .addSignatories(outputChatState.participants)
+                .toSignedTransaction()
+        }
+        val expectedMessage = "This method is not implemented for the mock ledger"
+        val actualMessage = exception.message
+        Assertions.assertTrue(actualMessage!!.contains(expectedMessage))
```

Comment:
Can you add the static imports so you don't need to call `Assertions.assertTrue` but simply `assertTrue` like you have in the Java template?
Same goes for `Assertions.assertThrow`

Reply:
Regpack test evidence:
![image](https://github.com/corda/CSDE-cordapp-template-kotlin/assets/140801590/e28e63e6-93fb-43c1-a5a0-680dbcf2a55e)